### PR TITLE
Adjust analyzer test with the latest changes

### PR DESCRIPTION
**Pull request by MDAlFattah**

### CODE REVIEW

File: release_tester/selenium_ui_test/pages/analyzers_page.py [hunk 1]
```diff
@@ -680,7 +679,7 @@ def add_new_analyzer(self, name, ui_data_dir=None):
             # ----------------------adding first pipeline analyzer as Norm analyzer--------------------------
             self.tprint(f'Selecting add analyzer button for {name} \n')
             if self.version_is_newer_than('3.11.99'):
-                add_analyzer01 = '//*[@id="chakra-modal--body-7"]/div/div[3]/div/button'
+                add_analyzer01 = "(//button[@class='chakra-button css-oni4cz'])[1]"
```
Comment (palashkaria) [inline release_tester/selenium_ui_test/pages/analyzers_page.py:682]: this `css-oni4cz` can change at any time. Let's add test IDs and use those here

File: release_tester/selenium_ui_test/pages/analyzers_page.py [hunk 2]
```diff
@@ -689,7 +688,7 @@ def add_new_analyzer(self, name, ui_data_dir=None):
 
             self.tprint(f'Selecting first pipeline analyzer as Norm for {name} \n')
             if self.version_is_newer_than('3.11.99'):
-                norm = '//*[@id="chakra-modal--body-7"]/div/div[3]/div/div/div[1]/div/div/div/div[1]/div[2]'
+                norm = "(//div[contains(@class,'css-nmh171')])[3]"
```
Comment (palashkaria) [inline release_tester/selenium_ui_test/pages/analyzers_page.py:691]: same as above, `css-XYZ` is not stable, can change on every build

Comment (MDAlFattah) [reply release_tester/selenium_ui_test/pages/analyzers_page.py]: Taken care of.

File: release_tester/selenium_ui_test/pages/analyzers_page.py (hunk 1 above)
Comment (MDAlFattah) [reply release_tester/selenium_ui_test/pages/analyzers_page.py]: Taken care of.

**Pull request merged**